procedures/meeting-observers.md

```diff
@@ -0,0 +1,39 @@
+# Meeting observers
+
+## Motivation
+
+There has been some interest in people joining meetings in a pure observational manner. (See [previous discussion on Zulip](https://rust-lang.zulipchat.com/#narrow/stream/392734-council/topic/Observing.20meetings.3F).) Further discussion on the specifics of a proposal happened on [Zulip](https://rust-lang.zulipchat.com/#narrow/stream/392734-council/topic/Meeting.20observers) and [in a council meeting](https://github.com/rust-lang/leadership-council/blob/245b22a105546a680aad582cf856311a80e93baf/minutes/sync-meeting/2023-07-20.md).
+
+## Process for meeting observers
+
+### Request for observation
+
+As an initial iteration, any Project member may join the public portion of council meetings to observe. No particular method of communication is required; those interested should either post in the [council zulip stream](https://rust-lang.zulipchat.com/#narrow/stream/392734-council), send an email to `council@rust-lang.org`, or (if necessary) message a council member. At least for now, non-Project members will not be admitted as meeting observers.
+
+### Meeting process
+
+When a meeting is scheduled, those who have requested to observe that meeting will get a calendar invite, in addition to the council. A minutes link will not be shared, but a public agenda link may be.
+
+During the meeting, observers should be mostly non-interactive, unless otherwise noted: no chat messages should be sent, and video and audio should be off. However, there may be times, like during check-ins and check-outs, where observers are encouraged to share their experience in chat. Observers should not record or screenshot the meting. If a meeting has a private session, then observers will either leave or be kicked.
+
+Intentional disruption of the meeting will result in an observer being kicked from the meeting and barred from future meetings.
```

Comment:
```suggestion
Intentional disruption of the meeting will result in an observer being kicked from the meeting and may result in the observer being barred from future meetings.
```

I'm slightly concerned this is moving towards mod team territory. How egregious of a disruption is needed? Would that disruption also imply a code of conduct issue? How long are they barred for? Is there a way to come back to the meetings? If someone is barred from future meetings, are they eligible to be made a Council member in the future?

It seems like we might be better saying we can kick out a disruptive member in the interest of keeping an effective meeting, and that depending on the behavior or if disruptiveness becomes a pattern then we will refer the issue to the mod team. We might want to reserve the right to kick people for _unintentional_ disruptions too. For example, they forget to mute and are banging pots and pans around in their kitchen and we can't contact them to mute them (this probably isn't the best example since you can generally mute other people in a call in this case). The high order bit here is to make sure we can keep an effective meeting, even if that means removing people who are detracting from that to an unreasonable extent, regardless of intent.

Reply:
This is reasonable. One qualification brought up in the meeting discussion that I didn't include here is admittance or Project members *in good standing*. Bringing that language back could be helpful as a means to bridge the gap. Regardless, I think its reasonable to bar people from joining that have been intentionally disruptive, regardless of referral to mod team.

I did originally include language re. unintentional disruption. I think was some comments of that being too strict.

Reply:
I've changed the wording to "may", but beyond that, I think we can adjust with experience if necessary.